app/assets/javascripts/discourse/app/components/glimmer-header/panel-wrapper.gjs

```diff
@@ -0,0 +1,9 @@
+const PanelWrapper = <template>
+  {{#if @panelElement}}
+    {{#in-element @panelElement}}
+      {{yield}}
+    {{/in-element}}
+  {{/if}}
+</template>;
```

Comment:
IIRC this is exactly what the existing `ConditionalInElement` component does in core? So maybe we just reuse that instead of making a separate `PanelComponent`?